stylesheets/_override.shame.scss

```diff
@@ -202,3 +202,10 @@
 .lt-ie10 .content-main {
     clear: both;
 }
+
+.icon--align-with-fields {
```

Comment:
Is this going to be/should this be resolved in a non-override way? Need a new ticket?